packages/vscode-mdx/package.json

```diff
@@ -25,8 +25,7 @@
   "main": "out/extension.js",
   "keywords": [
     "mdx",
-    "Markdown",
-    "Markdown React"
+    "markdown"
```

Comment:
These tags show up in the [marketplace](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx). There’s a max of 5 keywords. I don’t mind keeping `Markdown React`, but I dont value that specific keyword either. This field pretty arbritraty, I suggest other keyword suggestions. Should `MDX` be capitalized?